mypyc/test-data/run-strings.test

```diff
@@ -598,6 +600,7 @@ def test_decode() -> None:
     assert b.decode('gbk') == '浣犲ソ'
     assert b.decode('latin1') == 'ä½\xa0å¥½'
 
+[case testEncode]
```

Comment:
Here it would be reasonable to have them in a single test case, since these are related (all string operations) and having fewer run tests will speed up tests. But it's not a big deal. The original name (`testChrOrdEncodeDecode`) wasn't the clearest though. Also we already have `testStringOps`, which would cover these as well. Not sure what's the best way to organize our tests.